prereise/gather/griddata/hifld/data_access/load.py

```diff
@@ -0,0 +1,94 @@
+import json
+import os
+import tempfile
+from io import BytesIO
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import pandas as pd
+
+from prereise.gather.griddata.hifld.const import abv2state  # noqa F401
+
+
+def get_hifld_power_plants(path):
+    """Read the HIFLD Power Plants CSV file and keep operational plants located in
+    contiguous states.
+
+    :param str path: path to file. Either local or URL.
+    :return: (*pandas.DataFrame*) -- operational power plant in contiguous states.
+    """
+    data = pd.read_csv(path)
+    return data.query("STATUS == 'OP' and STATE in @abv2state")
+
+
+def get_hifld_generating_units(path):
+    """Read the HIFLD Generating Units CSV file and keep operational plants located in
+    contiguous states.
+
+    :param str path: path to file. Either local or URL.
+    :return: (*pandas.DataFrame*) -- operational generating units in contiguous states.
+    """
+    data = pd.read_csv(path)
+    return data.query("STATUS == 'OP' and STATE in @abv2state")
```

Comment:
These two functions seem to be the same except for the function names and doc strings?

Reply:
Yes, for now. We might have different filters once we are more familiar with the content of these files